plugins/rest/src/test/java/org/apache/struts2/rest/handler/JacksonXmlHandlerTest.java

```diff
@@ -60,10 +62,16 @@ public void testObjectToXml() throws Exception {
         // when
         Writer stream = new StringWriter();
         handler.fromObject(ai, obj, null, stream);
+        String actual = stream.toString();
```

Comment:
Wouldn't be better to move this line after `stream.flush()` on line 68?

Reply:
Yes you're right, I fixed the bug in my code. Is there any other improvement?

Reply:
That's all, thanks!